src/utils/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from './aadGroup';
```

Comment:
Just noticed I forgot to add the `planner utility` (#3268) to this file. Should I add it here right away or create a separate issue & PR for it?

Reply:
Good catch! Let's do it right away